lib/components/project/show.vue

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2017 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/opendatakit/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div>
+    <page-head v-show="project != null">
+      <template slot="title">
+        {{ project != null ? project.name : '' }}
+      </template>
+      <template slot="tabs">
+        <li :class="tabClass('')" role="presentation">
+          <router-link :to="tabPath('')">Overview</router-link>
+        </li>
+      </template>
+    </page-head>
+    <page-body>
+      <loading :state="awaitingResponse"/>
```

Comment:
`ProjectShow` does not send a request, but it still needs a loading message, because `ProjectHome` does not show a loading message. In other words, `ProjectShow` needs to show a loading message for the request for the project sent from `ProjectHome`. I've updated the logic accordingly.